cosmos-sdk-proto/Cargo.toml

```diff
@@ -17,7 +17,11 @@ rust-version = "1.72"
 
 [dependencies]
 prost = { version = "0.13", default-features = false }
+prost-types = "0.13"
 tendermint-proto = "0.39"
+serde = "1.0.203"
+pbjson = { git = "https://github.com/informalsystems/pbjson.git", package = "informalsystems-pbjson"  }
+pbjson-types = { git = "https://github.com/informalsystems/pbjson.git", package = "informalsystems-pbjson-types"  }
```

Comment:
Instead of these, use https://docs.rs/tendermint-proto/latest/tendermint_proto/google/protobuf/index.html

Reply:
Done, the extern_paths have been updated to point to the new tendermint rs version, and these deps have been removed